### PR TITLE
Save and reset button feature, start message

### PR DESCRIPTION
**Pull request by Hecktors**
Save button is disabled if use selection hasn't changed.
Reset button is disabled if no moves are selected.
Start message is displayed with first start of the app.

### CODE REVIEW

**Review by FabianAlleblas — COMMENTED** [review]
Niiiice 👍

File: src/components/MoveList.js [hunk 1]
```diff
@@ -12,30 +12,22 @@ MoveList.propTypes = {
...
     <li key={id}>
       <button
         data-testid="listitem-button"
-        onClick={() => openVideo(video_id, video_start)}
+        onClick={() => setVideo({ id: videoId, start: videoStart })}
       >
         {name}
-        {video_id && <img src={youTubeIcon} alt="youtube" />}
+        {videoId && <img src={youTubeIcon} alt="youtube" />}
         <span />
```
Comment (ninapeters) [inline src/components/MoveList.js:23]: What do you need the `span` for?

File: src/components/StartMessage.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      {isFirstAppStart ? (
+        <div className="first-start">
+          <div className="top">
+            <NorthIcon />
+            <br />
+            <span className="text">1. Choose the moves here.</span>
+          </div>
+          <div className="text">
+            Afer choosing at least 2 moves
+            <br /> you'll be ready to start
```
Comment (ninapeters) [inline src/components/StartMessage.js:17]: "you'll be ready to start."

File: src/components/StartMessage.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+          </div>
+
+          <div className="bottom">
+            <span className="text">2. Start the session here</span>
```
Comment (ninapeters) [inline src/components/StartMessage.js:21]: I would use punctuation marks consistently.

File: src/components/StartMessage.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+            <br />
+            <SouthIcon />
+          </div>
+        </div>
+      ) : (
+        <div className="default">
+          <span className="text">Select at least 2 moves</span>
```
Comment (ninapeters) [inline src/components/StartMessage.js:29]: Same here

File: src/components/StartMessage.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    </MessageStyled>
+  )
+}
+
+const MessageStyled = styled.div`
+  display: absolute;
+  text-align: center;
+  width: 100%;
+  height: 100%;
+  display: flex;
```
Comment (ninapeters) [inline src/components/StartMessage.js:41]: Can you assign display twice here. Is it really working?

File: src/styles/GlobalStyles.js [hunk 6]
```diff
@@ -7,9 +7,11 @@ export default createGlobalStyle`
...
         --color-button: #fff;
-        --color-disabled: #999;
-        --color-selected: #ff7109;
+        --color-disabled: #333;
+        --color-listitem: #999;
+        --color-listitem-active: #ff7109;
         --color-move: #ff7109;
+        --color-title: #ff7109;
         --color-warning: #d15353;
         --color-success: green;
```
Comment (ninapeters) [inline src/styles/GlobalStyles.js:16]: I would also define a hex code for the green tone.

**Review by ninapeters — COMMENTED** [review]
It's so interesting to watch your app grow. Keep it up! 👏

File: src/components/StartMessage.js (hunk 5 above)
Comment (Hecktors) [reply src/components/StartMessage.js:41]: No ☺️

File: src/components/StartMessage.js (hunk 2 above)
Comment (Hecktors) [reply src/components/StartMessage.js:17]: Going to change the text

File: src/components/MoveList.js (hunk 1 above)
Comment (Hecktors) [reply src/components/MoveList.js:23]: No. Not in use anymore

File: src/components/StartMessage.js (hunk 3 above)
Comment (Hecktors) [reply src/components/StartMessage.js:21]: You're right.

File: src/styles/GlobalStyles.js (hunk 6 above)
Comment (Hecktors) [reply src/styles/GlobalStyles.js:16]: It was kind of testing color. I already removed it.

**Pull request merged**